packages/jest-worker/src/base/__tests__/BaseWorkerPool.test.js

```diff
@@ -62,21 +62,6 @@ describe('BaseWorkerPool', () => {
     expect(pool.getWorkerById(3)).toBeDefined();
   });
 
-  it('creates and expoeses n workers', () => {
```

Comment:
Test above this one is exactly the same. Only difference is a typo in the name of this one.